routes/web.php

```diff
@@ -13,12 +15,45 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+// Route::get('/', function () {
```

Comment:
You can remove this if you don't want to use it

Reply:
Saye beki dh